### ISSUE TITLE
Various C++ errors when using pybombs on Ubuntu 16.04

### ISSUE THREAD

**Issue by adanowitz** [rwth-ti/gr-ofdm#2]
When installing this, I ran into several of each of the following types of errors:

src/gr-ofdm/lib/schmidl_tm_rec_stage2_impl.cc:85:29: error: invalid initialization of non-const reference of type ‘float&’ from an rvalue of type ‘float’
        float & y = acc1.imag();

allocation_src_impl.cc:201:78: error: cannot convert ‘zmq::socket_t’ to ‘void*’ in initialization
         zmq::pollitem_t items[] = { { *d_socket_feedback, 0, ZMQ_POLLIN, 0 } };

include/ofdm/ofdmi_mod.h:253:32: error: ‘constexpr’ needed for in-class initialization of static data member ‘const float m16qam_scheme::level_0’ of non-integral type [-fpermissive]
   static const float level_0 = 0.6325;

**Comment by johschmitz** [comment]
We need to look into this. I actually don't use pybombs, so I need some time to reproduce the error.
Meanwhile could you maybe  try to configure some less strict compiler flags and see if it goes through?

**Comment by adanowitz** [comment]
I managed to get it to compile by replacing the "float & y" with "const float & y", explicitly casting *d_socket_feedback as void* and changing "static const float" to "static constexpr float". I did try to compile on Ubuntu 17.04 using the download from github as-is and got the same error when using the build instructions on the readme. I believe CMAKE on my machine is using g++ 6.3.0 and gcc 6.3.0. I have not set any additional CMAKE system variables.
